src/main/java/uk/gov/hmcts/reform/preapi/entities/Case.java

```diff
@@ -48,6 +51,13 @@ public class Case extends CreatedModifiedAtEntity implements ISoftDeletable {
     @Transient
     private boolean isSoftDeleteOperation;
 
+    @Column(name = "state", nullable = false)
+    @JdbcTypeCode(SqlTypes.NAMED_ENUM)
+    private CaseState state;
```

Comment:
This should fix most of your errors from the tests
```suggestion
    private CaseState state = CaseState.OPEN;
```